qemu/tests/cfg/interrupt_check.cfg

```diff
@@ -19,7 +19,7 @@
                 - usb_device:
                     irq_pattern = "xhci_hcd"
                     usbs = usb_xhci
-                    usb_bus = "usb_xhci.0"
+                    usbdev_bus = "usb_xhci.0"
```

Comment:
@PaulYuuu  Seems this test was added by you, can you help review ? Thanks!

Reply:
It's good to me based on the avocado-vt patch.